### DISCUSSION TITLE
"Used by" section missing for my GitHub repo

### DISCUSSION THREAD

**Discussion by Tao-VanJS** [community/community#75445] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hi,

I recently noticed that the "Used by" section is missing in [my GitHub repo](https://github.com/vanjs-org/van) (it's used to be there). Could anyone let me know what could be the reason? Thank you so much!

FYI: the dependency of my repo can be found here: https://github.com/vanjs-org/van/network/dependents

**Comment by sky9262** [comment]
No Dependents:
The most straightforward reason could be that currently, there are no other repositories depending on your project. The "Used by" section will only appear if there are other GitHub repositories that have listed your repository as a dependency.

**Reply by Tao-VanJS** [reply]
Currently, there are 51 repositories depending on my project. You can refer to the list here: https://github.com/vanjs-org/van/network/dependents

**Reply by sky9262** [reply]
Check Repository Settings:
Double-check your repository settings to ensure that dependency tracking is enabled. Go to the repository settings, navigate to the "Security & analysis" section, and confirm that "Allow dependabot to update this repository's dependencies" is enabled.

**Reply by Tao-VanJS** [reply]
I found the section you mentioned:

> Allow Dependabot to open pull requests automatically to keep your dependencies up-to-date when new versions are available.

It looks like the option is to enable dependabot to open PRs for updating the dependencies of this particular project. I'm not sure whether this has anything to do showing the "Used by" section on the home page of the repo. Anyway, I just enabled this option. I will check if it can solve the problem.

**Reply by sky9262** [reply]
Please take a look at this docs (https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/exploring-the-dependencies-of-a-repository).

**Reply by Tao-VanJS** [reply]
Sorry, but I am not seeing the doc you pasted is relevant to the issue that I raised in this thread.

**Reply by nuxy** [reply]
> It looks like the option is to enable dependabot to open PRs for updating the dependencies of this particular project. I'm not sure whether this has anything to do showing the "Used by" section on the home page of the repo.

Requiring a dependabot access to create PRs just to get this working makes no sense.

What if the project is using a programming language the dependabot doesn't support?

**Reply by SiddhantSadangi** [reply]
https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/exploring-the-dependencies-of-a-repository#changing-the-used-by-package

**Comment by Tao-VanJS** [comment]
There are similar questions being asked. None of them seem to be answered: #74579, #73522.

**Reply by nuxy** [reply]
This question has been answered multiple times already since your initial post.

Please close this thread.

**Comment by nuxy** [comment, marked as answer]
Contacted support on 11/23 regarding this issue.  Their reply is below.

> Thank you for contacting GitHub support. GitHub recently changed it so the Used by section will only show on repositories where the selected package has at least 100 dependents, as documented [here](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/exploring-the-dependencies-of-a-repository#changing-the-used-by-package).

### References
- https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/exploring-the-dependencies-of-a-repository#changing-the-used-by-package

**Comment by Yashraj950** [comment]
Required More than 100 repositories depend on your package. to show used by 
https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/exploring-the-dependencies-of-a-repository#changing-the-used-by-package
